client/jetpack-cloud/sections/agency-signup/agency-signup-form/index.tsx

```diff
@@ -80,7 +80,11 @@ export default function AgencySignupForm() {
 	} );
 
 	useEffect( () => {
-		dispatch( recordTracksEvent( 'calypso_partner_portal_agency_signup_start' ) );
+		dispatch(
+			recordTracksEvent( 'calypso_partner_portal_agency_signup_start', {
+				formreferrer: referrer,
```

Comment:
After reviewing the code base, I noticed we typically use underscores to separate words in tracks events. Do you think it would be a good idea to follow that convention with form_referrer?

Reply:
You are right, there is a general naming convention (found in the Field Guide) covering properties as well, thank you.  I've also added 'source' to the property name too so it's slightly clearer what it refers to.